src/main/java/io/github/thunderz99/cosmos/condition/Queryable.java

```diff
@@ -0,0 +1,4 @@
+package io.github.thunderz99.cosmos.condition;
+
+public class Queryable {
```

Comment:
Please add comments for this. And Queryable should be an Interface. Not a public class.